Added the GogglesReminder plugin

Noatel opened the pull request:
So recently a friend of mine, created a ton of potions but later messaged me that he forgot his goggles after days of grinding it. This plugin prevents that, after creating a couple potions, a message (in red) pops up in chat saying "Don't forget your Prescription Goggles!". 


![image](https://github.com/user-attachments/assets/4956992f-76b8-4a49-aca8-5822f4519ece)

tylerwgrass commented:
- Your icon file should be at the root of your repo, named `icon.png` and no larger than 48x72px.
- Please delete the config file if you aren't using it yet. If you choose to add options in the future you can re-add it. 
- This looks like its going to trigger on any herblore xp (e.g. cleaning herbs) and not just potions which doesn't really make sense.

Noatel commented:
Hey! Thanks for you feedback 

- I have changed the image and put it into the root file
- Deleted the config file 
- Changed the logica to work only for potions based on the set i made in the class PotionItemIds
